include/cocaine/hpack/header.hpp

```diff
@@ -191,14 +191,18 @@ class header_t {
 };
 
 class header_storage_t {
+    header_t::zone_t zone;
```

Comment:
Personally I don't like this style, as in header public api of the class is more important than it's layout.